src/main/java/com/epam/spring/hw3/controller/DeliveryController.java

```diff
@@ -0,0 +1,46 @@
+package com.epam.spring.hw3.controller;
+
+import com.epam.spring.hw3.dto.OrderDTO;
+import com.epam.spring.hw3.service.OrderService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+public class DeliveryController {
+    private final OrderService orderService;
+
+    @ResponseStatus(HttpStatus.OK)
+    @GetMapping(value = "/orders")

```

Comment:
I would recommend to use @RequestMapping("/order") instead of duplicate same url in each method. And do not use "/orders" and "/order" in the same class, instead of "/orders" use "/order/all"